app/gateway/2.6.x/index.md

```diff
@@ -3,81 +3,119 @@ title: Kong Gateway
 subtitle: API gateway built for hybrid and multi-cloud, optimized for microservices and distributed architectures
 ---
 
-{{site.base_gateway}} is a lightweight, fast, and flexible cloud-native API gateway. It’s easy to download, install, and configure to get up and running once you know the basics. The gateway runs in front of any RESTful API and is extended through modules and plugins which provide extra functionality beyond the core platform.
+{{site.base_gateway}} is a lightweight, fast, and flexible cloud-native API
+gateway. The gateway runs in front of any RESTful API and can be extended through
```

Comment:
```suggestion
gateway. The Gateway runs in front of any RESTful API and can be extended through
```

I have been capitalizing Gateway. Checked out style guide - let's add whether or not to capitalize Gateway when it's not proceeded by "Kong".